src/dbnode/server/server.go

```diff
@@ -682,6 +696,13 @@ func Run(runOpts RunOptions) {
 			}
 
 			bs.SetBootstrapperProvider(updated.BootstrapperProvider())
+
+			if err := bsGauge.UpdateStringList(bootstrappers); err != nil {
+				logger.Error("unable to update bootstrap gauge with new bootstrappers",
+					zap.String("bootstrappers", fmt.Sprintf("%#v", bootstrappers)),
```

Comment:
You can use `zap.Strings("foo", values)` here instead of a sprintf I believe?

Reply:
Fixed